samples/sensor/lsm6dsl/README.rst

```diff
@@ -0,0 +1,64 @@
+.. _lsm6dsl:
+
+LSM6DSL: IMU sensor Monitor
+###########################
+
+Overview
+********
+This sample sets the LSM6DSL accelerometer and gyroscope to 104Hz
+and enable a trigger on data ready. It displays on the console the
+values for accelerometer and gyroscope, plus optionally the values of
+any magnetometer or pressure sensor attached to it (sensorhub function).
+
+
+Requirements
+************
+
+This sample uses the LSM6DSL sensor controlled using the I2C or SPI interface.
+It has been tested on both :ref:`96b_argonkey` and disco_l475_iot1 board.
+
+References
+**********
+
+ - LSM6DSL http://www.st.com/en/mems-and-sensors/lsm6dsl.html
```

Comment:
Rendering is better without the leading space:
```suggestion
- LSM6DSL http://www.st.com/en/mems-and-sensors/lsm6dsl.html
```

Reply:
ack